index.html

```diff
@@ -236,12 +236,12 @@ <h2>About<br> Myself </h2>
                <div class="frame_skills_bar">
                   <div class="frameworks_bar">
                      <div class="frameworks">Frameworks</div>
-                     <div> <img src="images/arrow2/Normal Button/Tertiary/Icons/Union.png" alt=""></div>
+                     <div> <img src="images/arrow2/Normal Button/Tertiary/Icons/Union.png" alt="My frameworks"></div>
                   </div>
                   <hr>
                   <div class="skills_bar">
                      <div class="skills">Skills</div>
-                     <div> <img src="images/arrow2/Normal Button/Tertiary/Icons/Union.png" alt=""></div>
+                     <div> <img src="images/arrow2/Normal Button/Tertiary/Icons/Union.png" alt="My skills"></div>
                   </div>
```

Comment:
please consider wrapping all the **li tag** with **ul tage** to make it semantically correct 👍